proposals/003-installer-guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Introduction
+
+The Haskell Foundation would like see a central Haskell download page in which the Haskell curious
+could easily download and run a installer to provide a standard Haskell environment that supports
+the principal build tools (`cabal-install` and `stack`) on the main platforms (Linux, macOS and
```

Comment:
Also: `ghc`.  I think it's critical that someone who has installed haskell can run `ghci` and play around at the repl, for example.  Not all Haskell users are building projects and need project-focused build tools.